src/App.jsx

```diff
@@ -1,14 +1,38 @@
+/* eslint-disable no-lone-blocks */
+/* eslint-disable no-undef */
+/* eslint-disable no-unused-vars */
+/* eslint-disable prettier/prettier */
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
```

Comment:
The import `usersFromServer` is used in the code, so this line is fine. Ensure that the JSON file path is correct and the data structure matches the expected format.